Index.html

```diff
@@ -0,0 +1,282 @@
+            </li>
+          </ul>
+        </main>
+      </section>
+      <section>
+        <main>
+          <h1 class="headline-2">My Recent Works</h1>
+          <hr class="h-row" />
+          <div class="img-1"></div>
+          <h1 class="headline-3">Multi-Post Stories</h1>
+          <p class="text-content-1">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="btn-list">
+            <li>
+              <button class="btn-css"><a href="#">CSS</a></button>
+            </li>
+            <li>
+              <button class="btn-html"><a href="#">HTML</a></button>
+            </li>
+            <li>
+              <button class="btn-bootstrap"><a href="#">BOOTSTRAP</a></button>
+            </li>
+            <li>
+              <button class="btn-ruby"><a href="#">RUBY</a></button>
+            </li>
+          </ul>
+          <button class="btn-sub-1"><a href="#">See project</a></button>
+          <div class="pro-art">
+            <h1 class="headline-4">Profesional Art Printing Data</h1>
+            <p class="text-content-2">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="btn-list-1">
+              <li>
+                <button class="btn-html-1"><a href="#">HTML</a></button>
+              </li>
+              <li>
+                <button class="btn-bootstrap-1">
+                  <a href="#">BOOTSTRAP</a>
+                </button>
+              </li>
+              <li>
+                <button class="btn-ruby-1"><a href="#">RUBY</a></button>
+              </li>
+            </ul>
+            <button class="btn-sub-2"><a href="#">See project</a></button>
+          </div>
+          <div class="pro-art">
+            <h1 class="headline-4">Profesional Art Printing Data</h1>
+            <p class="text-content-2">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="btn-list-1">
+              <li>
+                <button class="btn-html-1"><a href="#">HTML</a></button>
+              </li>
+              <li>
+                <button class="btn-bootstrap-1">
+                  <a href="#">BOOTSTRAP</a>
+                </button>
+              </li>
+              <li>
+                <button class="btn-ruby-1"><a href="#">RUBY</a></button>
+              </li>
+            </ul>
+            <button class="btn-sub-2"><a href="#">See project</a></button>
+          </div>
+          <div class="pro-art">
+            <h1 class="headline-4">Profesional Art Printing Data</h1>
```

Comment:
- [ ] `h1` tag is for the main title `Hey There, I'm Yaser.` And `h1` tag only appears one time per webpage. It would be better to use other `hx` tags and maintain the hierarchy.  Kindly fix this issue.